### PR TITLE
CNV-3133: VM dashboard

### PR DESCRIPTION
**Pull request by ousleyp**
Jira: https://issues.redhat.com/browse/CNV-3133

For CNV 2.2/enterprise-4.3 only. Peer review complete 1/6.

Preview build (ignore date): http://file.bos.redhat.com/pousley/010520/cnv3133-vmdash/cnv/cnv_users_guide/cnv-viewing-information-about-vm-workloads.html

### CODE REVIEW

File: modules/cnv-about-the-vm-dashboard.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-viewing-information-about-vm-workloads.adoc
+
+[id="cnv-about-the-vm-dashboard_{context}"]
+= About the Virtual Machines dashboard
+
+Access the *Virtual Machines* dashboard from the {product-title} web console by navigating to the *Workloads* -> *Virtual Machines* page.
```
Comment (mareklibra) [inline modules/cnv-about-the-vm-dashboard.adoc]: In fact, a VM needs to be selected. The `*Workloads* -> *Virtual Machines* page` shows list of (all|namespaced) VMs, a VM needs to be clicked to see it's details. The dashboard is the default landing page (means subtab) for the VM detail.

**Review by mareklibra — COMMENTED** [review]
I have small comment regarding navigation, otherwise lgtm.

File: modules/cnv-about-the-vm-dashboard.adoc (hunk 1 above)
Comment (ousleyp) [reply modules/cnv-about-the-vm-dashboard.adoc]: @mareklibra I was wondering how you'd choose the right VM and forgot to ask! Thank you. Does this sound sufficient? 

`Access the *Virtual Machines* dashboard from the {product-title} web console by navigating to the *Workloads* -> *Virtual Machines* page and clicking a virtual machine.`

File: modules/cnv-about-the-vm-dashboard.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+[id="cnv-about-the-vm-dashboard_{context}"]
+= About the Virtual Machines dashboard
+
+Access the *Virtual Machines* dashboard from the {product-title} web console by navigating to the *Workloads* -> *Virtual Machines* page and
+clicking the name of a virtual machine.
+
+The dashboard includes the following cards:
+
+* *Details* provides identifying information about the virtual machine, including:
+** the name of the virtual machine
```
Comment (aburdenthehand) [inline modules/cnv-about-the-vm-dashboard.adoc]: Nit: I think the points should all have capitals.

File: modules/cnv-about-the-vm-dashboard.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+** the namespace
+** when the virtual machine was created
```
Comment (aburdenthehand) [inline modules/cnv-about-the-vm-dashboard.adoc]: Nit: Would prefer this to be 'date' or 'time' of virtual machine creation

File: modules/cnv-about-the-vm-dashboard.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+** the node name
+** the virtual machine's IP address
```
Comment (aburdenthehand) [inline modules/cnv-about-the-vm-dashboard.adoc]: IBM Style (p41) "Avoid using ’s to show the possessive form of inanimate objects. Make the possessive
noun an adjective, or use the possessive noun in a prepositional phrase."

**Comment by aburdenthehand** [comment]
3 nits, otherwise lgtm!

File: modules/cnv-about-the-vm-dashboard.adoc (hunk 3 above)
Comment (ousleyp) [reply modules/cnv-about-the-vm-dashboard.adoc]: Good point. I made the whole section more minimalistic and it's now just 'Date of creation'

File: modules/cnv-about-the-vm-dashboard.adoc (hunk 4 above)
Comment (ousleyp) [reply modules/cnv-about-the-vm-dashboard.adoc]: Good reminder :)

**Pull request merged**